templates/queue.html

```diff
@@ -58,6 +58,8 @@ <h1>
         {% match pr.queue_status() %}
         {% when Approved(_) %}
         approved
+        {% when ReadyForMerge(_, _) %}
+        approved
```

Comment:
This state would only be visible for a few seconds usuallly, but I think that we could actually show it, for easier debugging. So maybe `ready for a merge`?